inst/pages/21_microbiome_community.qmd

```diff
@@ -94,7 +100,8 @@ ggplot(df, aes(x = SampleID, y = FeatureID, fill = clr_z)) +
   labs(x = "Samples", y = "Taxa")
 ```
 
-_pheatmap_ is a package that provides methods to plot clustered heatmaps. 
+*pheatmap* is a package that provides methods to plot clustered
+heatmaps.
```

Comment:
@antagomir Do we prefer pheatmap or ComplexHeatmap? I would prefer ComplexHeatmap as you can do much more with it. In fact, you can also create pheatmap-like plots with ComplexHeatmap::pheatmap()

Reply:
ok to me.